README.md

```diff
@@ -1,2 +1,88 @@
-# JS-restaurant-page
+# Js-Restaurant-page
+
+## Table of contents
+
+- [About](#about)
+- [Features](#features)
+- [Live Demo](#live-demo)
+- [Built with](#built-with)
+- [Getting started](#getting-started)
+  * [Installation instructions](#installation-instructions)
+- [Author](#author)
+- [Show your support](#show-your-support)
+- [Acknowledgements](#acknowledgments)
+- [License](#-license)
+
+---
+
+## About
+
 Restaurant page with the pages build entirely with JavaScript and Webpack for modularization
+
+![Screenshot-main-page](dist/assets/images/Garys-bistro-shot.png)
+
+
+## Features
+
+The features of this project are, user can:
+
+1. page change dynamically
+2. Javascript modules.
+
+
+## Live Demo
+
+This project is hosted on heroku.
+
+[Live demo link](https://helciodev.me/JS-restaurant-page/)
+
+
+## Built With
+
+- [![forthebadge](https://forthebadge.com/images/badges/made-with-javascript.svg)](https://forthebadge.com)
+- [![forthebadge](https://forthebadge.com/images/badges/uses-html.svg)](https://forthebadge.com)
+- [![forthebadge](https://forthebadge.com/images/badges/uses-css.svg)](https://forthebadge.com)
+
+## Getting Started
+
+
+### Installation instructions
```

Comment:
- Add instructions for installing packages, building the app locally and running